packages/react-storage/src/components/StorageImage/types.ts

```diff
@@ -15,6 +16,7 @@ export interface StorageImageProps extends Omit<ImageProps, 'src'> {
    * `accessLevel` will be replaced with `path` in a future major version of Amplify UI. See https://ui.docs.amplify.aws/react/connected-components/storage/storageimage#props
    */
   accessLevel: StorageAccessLevel;
+  bucket?: StorageBucket;
```

Comment:
Do we `bucket` here if it's not supported in Gen1?

Reply:
No, we shouldn't. This was a misunderstanding on my part